nginx-proxy-manager/docker-compose.yml

```
@@ -0,0 +1,22 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: nginx-proxy-manager_web_1
+      APP_PORT: 81
+  web:
+    image: jc21/nginx-proxy-manager:2.11.3@sha256:5bd2aae54ac0b6f4ef6777143ab736baf5366807894a4c7fca7bfa2fbaf1489c
+    hostname: nginx-proxy-manager_web_1
+    restart: on-failure
+    ports:
+      - 443:443
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+      - ${APP_DATA_DIR}/letsencrypt:/etc/letsencrypt
```

Let's put both of these inside the `${APP_DATA_DIR}/data` directory. So something like:
```suggestion
    volumes:
      - ${APP_DATA_DIR}/data/npm-data:/data
      - ${APP_DATA_DIR}/data/letsencrypt:/etc/letsencrypt
```

You'll need to make the equivalent changes to the subdirectories committed in this PR